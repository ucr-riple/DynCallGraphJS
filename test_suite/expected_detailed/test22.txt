{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test22.js:8:15:8:33)': [
        'create (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test22.js:8:15:8:33)'
    ],
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test22.js:9:1:9:22)': [
        'showDetails (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test22.js:3:16:5:4)'
    ]
}