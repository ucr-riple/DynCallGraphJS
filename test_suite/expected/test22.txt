{ '(test22.js:8:15:8:33)': [ 'create (Native)' ],
  '(test22.js:9:1:9:22)':
   [ '(test22.js:3:16:5:4)' ] }